controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -61,6 +61,10 @@ static bool validate_hex_notation(const std::string &str)
         return false;
     }
 
+    if (expected_octets != 0 && (str.size() - 2) / 2 != expected_octets) {
```

Comment:
will add

Reply:
Now I understand but in that case, why not use `sizeof("0x")`?
Regarding the division by 2 - still not clear to me. One octet == one byte, so why 2?

Reply:
sizeof("0x") is 3 because it add '\0'.
I can create const string do that:
```
const std::string prefix("0x");
(str.size() - prefix.size()) /2
```

> Regarding the division by 2 - still not clear to me. One octet == one byte, so why 2?
because one byte is represented by to characters. 
For example, "FF" is stored in 1 byte, but has to chars 'F' and 'F'.

Reply:
I don't think the std::string("0x").size() makes things much clearer, I prefer the comment. It was anyway already clear to me without comment :-)